sematic/ui/packages/main/src/pipelines/RunMetricsPanel.tsx

```diff
@@ -0,0 +1,103 @@
+import { Timeline } from "@mui/icons-material";
```

Comment:
Maybe have the imports sorted.

Reply:
I don't know how to do that.

Reply:
If you use VS Code, you can either use the command "Organize Imports" or use its shortcut:

<img width="745" alt="image" src="https://github.com/sematic-ai/sematic/assets/133257643/bc6e183c-2b07-48ab-a5ea-726cf0f8c072">